packages/api/amplify_api/test/connectivity_plus_platform_test.dart

```diff
@@ -14,8 +14,9 @@
 
 import 'dart:async';
```

Comment:
Let's rewrite tests to no longer use `ConnectivityPlatform.instance` for mocking. We should be dependency injecting our own `ConnectivityPlatform` everywhere now.

Reply:
I agree. I added a TODO comment for that here partially bc I wanted to keep this PR to file re-arranging as much as possible and not refactor things here.